src/main/scala/org/neo4j/spark/util/Neo4jImplicits.scala

```diff
@@ -56,4 +58,26 @@ object Neo4jImplicits {
       (entityFields ++ entityMap).asJava
     }
   }
+
+  implicit class FilterImplicit(filter: Filter) {
+    def getAttribute: Option[String] = Option(filter match {
+      case eqns: EqualNullSafe => eqns.attribute
+      case eq: EqualTo => eq.attribute
+      case gt: GreaterThan => gt.attribute
+      case gte: GreaterThanOrEqual => gte.attribute
+      case lt: LessThan => lt.attribute
+      case lte: LessThanOrEqual => lte.attribute
+      case in: In => in.attribute
+      case notNull: IsNotNull => notNull.attribute
+      case isNull: IsNull => isNull.attribute
+      case startWith: StringStartsWith => startWith.attribute
+      case endsWith: StringEndsWith => endsWith.attribute
+      case contains: StringContains => contains.attribute
```

Comment:
what about list data type (IN) and boolean values (no comparion where f.flag ) and temporal / spatial values (distance, components, ...)? perhaps in a later PR?

Reply:
> what about list data type (IN)

these are already handled (see line 72 and [Neo4jUtil:228](https://github.com/neo4j-contrib/neo4j-spark-connector/pull/162/files/ede9ef43dceeab9fbb86e7ceee96f45a6f6824aa#diff-f4be0e8f0dc4f5a8b2cd064a79a7461bR228)

> boolean values (no comparion where f.flag )
Spark doesn't flag boolean filtering "Neo4j style" but needs to be explicit: `f.flag = true`